.github/workflows/container-image.yml

```diff
@@ -1,8 +1,8 @@
 name: Build porta container image
 on:
-  push:
-    branches:
-      - master
+  # push:
+  #   branches:
+  #   - master
```

Comment:
Why is this commented out?

Reply:
Because I'm not yet sure we want to have it build for every push to master. Or just have ability to trigger manually for whatever branch we want only when we need.